### PR TITLE
comply with stricter cpplint rules

### PR DESCRIPTION
**Pull request by dirk-thomas**
Update style to pass new cpplint check (ament/ament_lint#59).

http://ci.ros2.org/job/ci_linux/1763/

### CODE REVIEW

File: intra_process_demo/src/image_pipeline/camera_node.cpp [hunk 1]
```diff
@@ -16,6 +16,8 @@
 
 #include <rclcpp/rclcpp.hpp>
```
Comment (Karsten1987) [inline intra_process_demo/src/image_pipeline/camera_node.cpp]: Can we be consistent on the order and how to include the rcl headers? see [this demo](https://github.com/ros2/demos/blob/master/intra_process_demo/src/cyclic_pipeline/cyclic_pipeline.cpp#L20) for comparison

Comment (dirk-thomas) [inline intra_process_demo/src/image_pipeline/camera_node.cpp]: I agree that having `memory` further up in the list is preferred. But in some cases (e.g. https://github.com/ros2/demos/pull/80/files#diff-c53d060e7c287a653e8fef96c7f7a4eeR33) that would require quite a few of includes to be changed from `<...>` to `"..."`. Not sure if we want to do that?

Comment (wjwwood) [inline intra_process_demo/src/image_pipeline/camera_node.cpp]: I'd vote to merge as is and ticket the annoyance. For the long term solution I'd like to see cpplint.py patched to allow something like this:

```c++
#include <memory>

#include <rclcpp/rclcpp.hpp>

#include "foo/foo.hpp"
```

Comment (dirk-thomas) [inline intra_process_demo/src/image_pipeline/camera_node.cpp]: Done: ament/ament_lint#62

**Pull request merged**